schemas/primary_diagnosis.json

```diff
@@ -50,10 +50,11 @@
       "description": "The code to represent the cancer type using the WHO ICD-10 code (https://icd.who.int/browse10/2016/en#/) classification.",
       "restrictions": {
         "required": true,
-        "regex": "[A-Z]{1}[0-9]{2}.[0-9]{0,3}[A-Z]{0,1}$"
+        "regex": "\\bC[0-9]{2}.[0-9]{0,3}[A-Z]{0,1}$"
```

Comment:
use `^` for start of string. `\b` will probably work in this context, but technically implies start of word not start of the whole string.

Reply:
I'll get back to you as soon as I confirm this